spec/koans.js

```diff
@@ -85,54 +94,55 @@ describe('`string.includes()` finds string within another string. ', () => {
 
   describe('find a single character', function() {
     it('in a three char string', function() {
-      /* const searchString = ???? */
-      //expect('xyz'.includes(searchString)).toBe(true);
+      const searchString = "yz";
+      expect('xyz'.includes(searchString)).toBe(true);
```

Comment:
this one was about finding a single character, (const searchString = "y";), but it doesn't change much !